File: host/src/main/java/com/example/testapp/SplashActivity.java
package com.example.testapp;

import android.app.Activity;
import android.content.Intent;
import android.os.Bundle;

import androidx.annotation.Nullable;

import com.example.testapp.activities.DampLayoutActivity;

public class SplashActivity extends Activity {

    private String tag = "SplashActivity";

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        getWindow().getDecorView().setBackground(getResources().getDrawable(R.drawable.gal));
        setContentView(R.layout.activity_splash);
        startActivity(new Intent(this, DampLayoutActivity.class));
        finish();
    }

}

File: host/src/main/java/com/example/testapp/SplashActivity.kt
package com.example.testapp

import android.os.*
import androidx.appcompat.app.AppCompatActivity
import android.util.Log
import kotlinx.android.synthetic.main.activity_splash.*

class SplashActivity : AppCompatActivity() {

    private val tag = "SplashActivity"
    private val handlerThread = HandlerThread("worker")

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splash)

        handlerThread.start()

        val handlerB = Handler(handlerThread.looper)


        button.setOnClickListener {



        }
    }


    class CusThread : Thread() {
        override fun run() {
            super.run()

            while (true) {
                sleep(800)
                Log.i(tag, "do something...")
            }
        }
    }

}
